test/controllers/update/update.review.statement.controller.spec.ts

```diff
@@ -0,0 +1,181 @@
+mockCloseTransaction.mockReturnValue( TRANSACTION_CLOSED_RESPONSE );
+
+const mockPaymentsSession = startPaymentsSession as jest.Mock;
+mockPaymentsSession.mockReturnValue( "CONFIRMATION_URL" );
+
+describe("Update review overseas entity information controller tests", () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockGetApplicationData.mockReturnValue({
+      ...APPLICATION_DATA_CH_REF_UPDATE_MOCK,
+    });
+  });
+
+  describe("GET tests", () => {
+    test(`renders the ${UPDATE_REVIEW_STATEMENT_PAGE} page`, async () => {
+      mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_UPDATE_BO_MOCK);
+      const resp = await request(app).get(UPDATE_REVIEW_STATEMENT_URL);
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_PAGE_TITLE);
+      expect(resp.text).toContain(UPDATE_CHECK_YOUR_ANSWERS_CONTACT_DETAILS);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_WHO_CAN_WE_CONTACT);
+      expect(resp.text).toContain(CONTINUE_BUTTON_TEXT);
+    });
+
+    test(`renders the ${UPDATE_REVIEW_STATEMENT_PAGE} page with contact details section with (ceased) existing BO`, async () => {
+      mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_CH_REF_UPDATE_MOCK);
+      const resp = await request(app).get(UPDATE_REVIEW_STATEMENT_URL);
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_PAGE_TITLE);
+      expect(resp.text).toContain(UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL);
+      expect(resp.text).toContain(UPDATE_CHECK_YOUR_ANSWERS_CONTACT_DETAILS);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_WHO_CAN_WE_CONTACT);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_BENEFICIAL_OWNER_STATEMENT);
+      expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_BENEFICIAL_OWNER_STATEMENTS_CEASED_TITLE);
+      expect(resp.text).toContain(UPDATE_PRESENTER_CHANGE_FULL_NAME);
+      expect(resp.text).toContain(UPDATE_PRESENTER_CHANGE_EMAIL);
+    });
+
+    test(`renders the ${UPDATE_REVIEW_STATEMENT_PAGE} page overseas entity details section`, async () => {
+      const resp = await request(app).get(UPDATE_REVIEW_STATEMENT_URL);
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(OVERSEAS_ENTITY_SECTION_HEADING);
+      expect(resp.text).toContain("Overseas Entity Name");
+      expect(resp.text).toContain("SA000392");
+    });
+
+    test(`renders the ${UPDATE_REVIEW_STATEMENT_PAGE} page beneficial owner section is not rendered if no BO data`, async () => {
+      mockGetApplicationData.mockReturnValue({
+        ...APPLICATION_DATA_UPDATE_NO_BO_OR_MO_TO_REVIEW,
+      });
+
+      const resp = await request(app).get(UPDATE_REVIEW_STATEMENT_URL);
+      expect(resp.status).toEqual(200);
+      expect(resp.text).not.toContain(BENEFICIAL_OWNER_HEADING);
+      expect(resp.text).not.toContain("Beneficial owners you have reviewed");
```

Comment:
Not sure if this exists on the prototype?

Reply:
Reviewed section section does not exist on the prototype (Personally will prefer to take them out) but it is in the dev notes on the ticket. MO Sections also not part of the prototype. I guess it was decided when the ticket was refined?

Reply:
Prototype has been updated to include MO section